cli/src/command/account.rs

```diff
@@ -838,3 +843,46 @@ async fn print_address(account_handle: &AccountHandle, address: &AccountAddress)
 
     Ok(())
 }
+
+async fn print_claimable_outputs(account_handle: &AccountHandle, balance: &AccountBalance) -> Result<(), Error> {
+    println_log_info!("Claimable outputs: {}", balance.potentially_locked_outputs().len());
+
+    for output_id in balance
+        .potentially_locked_outputs()
+        .iter()
+        .filter_map(|(output_id, unlockable)| unlockable.then_some(output_id))
+    {
+        if let Some(output_data) = account_handle.get_output(output_id).await {
+            let output = output_data.output;
+            let kind = match output {
+                Output::Nft(_) => "Nft",
+                Output::Basic(_) => "Basic",
+                _ => unreachable!(),
+            };
+            println_log_info!("- output: {output_id} ({kind})");
```

Comment:
Personal preference but I would remove output and debug output_id instead of display it, more consistent and less repetitive

Reply:
okay ... but my view on this is that debug is for ... well debugging and display for ... displaying something (to the user). It may be more consistent atm but I wonder if we can slowly but surely change that. When we show something to the user it *should* be the Display implementation, not the Debug.

Reply:
I disagree, Debug/Display is contextual. What you are currently doing now is basically the same as what OutputId Debug is doing but in a longer and inconsistent way. There is no point saying `- output` as each line, we know it's outputs, we just said it.

Reply:
I am not arguing to keep the `- output`